### ISSUE TITLE
Mouse motion closes a ToolTip shown by keyboard

### ISSUE THREAD

**Issue by SamBent** [dotnet/wpf#4616]
* .NET Core Version: 3.0+
* Windows version: all
* Does the bug reproduce also in WPF for .NET Framework 4.8?: Yes
* Is this bug related specifically to tooling in Visual Studio (e.g. XAML Designer, Code editing, etc...)? no
* Security issues and bugs should be reported privately, learn more via our [responsible disclosure guidelines](https://github.com/dotnet/wpf/blob/master/README.md#reporting-security-issues-and-security-bugs).

 
 
 **Problem description:**
1. Run sample app from #4612
2. Move the mouse over the "Show Ribbon" checkbox (or any element that doesn't have a ToolTip)
3. Press TAB repeatedly to move focus to the second Image (look for the focus adorner)
4. Wait for the Image ToolTip to appear
5. Move the mouse away from the checkbox.
 
 **Actual behavior:** 
ToolTip disappears

 **Expected behavior:**
 ToolTip displays (for 15 seconds)

When a tooltip is shown because of keyboard focus, unrelated mouse motion should not hide it.

**Comment by SamBent** [comment]
Fixed by the tooltip work in 6.0-preview7.
